### ISSUE TITLE
Add warning dialogue for "Remove this track" button in song editor.

### ISSUE THREAD

**Issue by coderman64** [LMMS/lmms#5263]
I have noticed, while using the current release version of LMMS, it is incredibly easy to accidentally hit "Remove this track" when you mean to hit "Clone this track." In other applications, permanent operations like this would require a dialogue to make sure the user actually wants the track to be deleted and didn't click it by mistake. I propose a similar dialogue be implemented in LMMS, somewhat akin to the save/discard dialogue that appears when you try to close a project with unsaved changes:
![image](https://user-images.githubusercontent.com/12971494/67178174-c4e1e900-f39f-11e9-97dc-020b284743b0.png)
Quick-and-dirty GIMP mockup:
![image](https://user-images.githubusercontent.com/12971494/67178332-505b7a00-f3a0-11e9-9cdf-6ed4854a183f.png)

**Comment by itlives** [comment]
If this is still open ill gladly take it. I used to contribute on lmms a while ago but I had to pause because of my job.

**Comment by zonkmachine** [comment]
@fschuelke Please fix this! :whale:

**Comment by musikBear** [comment]
I have inserted a response control through a messagebox: 
![image](https://user-images.githubusercontent.com/6662066/82563847-0f1a8b00-9b78-11ea-9d0e-eda719c6df49.png)
This can be closed.

**Comment by Spekular** [comment]
> This can be closed.

Not until you've opened a PR that's been reviewed, approved, and merged. For starters it should say "Removing a track can't be *undone*." and I would prefer the OP's suggested "Are you sure you want to remove [track name]?" over "Do you want to proceed".

Secondly, I would personally appreciate if this dialogue could be disabled in settings or via a "don't ask me again" checkbox as it's never been an issue for me. (If it really is a common issue then the layout of the menu ought to change to prevent this, but that's a separate issue and I'm fine with this solution if it's optional)

**Comment by zonkmachine** [comment]
Thirdly there is a PR adressing this already and I was just about to bump it.
https://github.com/LMMS/lmms/pull/5274

**Comment by musikBear** [comment]
:p so 
![image](https://user-images.githubusercontent.com/6662066/82620213-343ee600-9bd8-11ea-93ce-91c43851272a.png) Means *'done and in PR'*, i only looked at the date ..nov 2019..
Nwm i thought myself usage of QmessageBox, through input of a user' response 
Time well spent anywitch

**Comment by coderman64** [comment]
Sorry about the confusion, but to be fair, fschuelke's PR is linked right above your first comment.

**Comment by musikBear** [comment]
@coderman64 i see it now, but not earlier, i am partly blind -no joke 🦇
